vcdh5cc.l10n

```diff
@@ -0,0 +1,27 @@
+#
+# This file lists all files that should be localized for a release.
+# The file syntax and the 'l10n-util' command are described on the Wiki page
+#
+#    https://wiki.eng.vmware.com/Build/L10NSupport
```

Comment:
no one outside VMware has access to this internal site

Reply:
I've spoken with the l10n team and we can remove all the extra comments. 

Just add a few lines of our own saying that these translations are maintained by VMware internally and this file contains all the locations in this repo that must be localized